library/packages/src/modules/SignatureCheckDialogs.rb

```diff
@@ -878,10 +876,10 @@ def HandleDoNotShowDialogAgain(default_return, dont_show_dialog_ident, dont_show
 
     # Function adds delimiter between after_chars characters in the string
     #
-    # @param string to be splitted
-    # @param [String] delimiter
-    # @param integer after characters
-    # @return [String] with delimiters
+    # @param whattosplit [String] text to be splitted
```

Comment:
np: Here we're using the form "param_name [class] description". However, we're using "[class] param_name description" in [other](https://github.com/yast/yast-yast2/pull/565/files#diff-02086261e6ecff029ebc83758fdf1b6fR474) [places](https://github.com/yast/yast-yast2/pull/565/files#diff-87d364fd10564ca337c52efbb988328cR160). I prefer this one (param name first) but I'm ok with both of them.